cli/determined_cli/experiment.py

```diff
@@ -492,15 +492,15 @@ def list_trials(args: Namespace) -> None:
     r = api.get(args.master, "experiments/{}/summary".format(args.experiment_id))
     experiment = r.json()
 
-    headers = ["Trial ID", "State", "H-Params", "Start Time", "End Time", "# of Steps"]
+    headers = ["Trial ID", "State", "H-Params", "Start Time", "End Time", "# of Batches"]
     values = [
         [
             t["id"],
             t["state"],
             json.dumps(t["hparams"], indent=4),
             render.format_time(t["start_time"]),
             render.format_time(t["end_time"]),
-            t["num_steps"],
+            t["total_batches_processed"],
```

Comment:
`total_batches_processed` on the trial is the actual total (done as a sum agg in postgres).